File: TransCoder_tokenized_test_set_functions/python/SUM_TWO_LARGE_NUMBERS_1.py
def findSum ( str1, str2 ) :
    if len ( str1 ) > len ( str2 ) :
        temp = str1
        str1 = str2
        str2 = temp
    str3 = ""
    n1 = len ( str1 )
    n2 = len ( str2 )
    diff = n2 - n1
    carry = 0
    for i in range ( n1 - 1, - 1, - 1 ) :
        sum = ( ( ord ( str1 [ i ] ) - ord ( '0' ) ) + int ( ( ord ( str2 [ i + diff ] ) - ord ( '0' ) ) ) + carry )
        str3 = str3 + str ( sum % 10 )
        carry = sum //10
    foriinrange(n2-n1-1,-1,-1):
        sum=((ord(str2[i])-ord('0'))+carry)
        str3=str3+str(sum%10)
        carry=sum//10
    if(carry):
        str3+str(carry+'0')
    str3=str3[::-1]
    returnstr3

File: results/py/SUM_TWO_LARGE_NUMBERS_1.cpp
string f_filled ( string str1 , string str2 ) {
  if ( str1 . length ( ) > str2 . length ( ) ) {
    string temp = str1 ;
    str1 = str2 ;
    str2 = temp ;
  }
  string str3 ;
  int n1 = str1 . length ( ) ;
  int n2 = str2 . length ( ) ;
  int diff = n2 - n1 ;
  int carry = 0 ;
  for ( int i = n1 - 1 ;
  i >= 0 ;
  i -- ) {
    int sum = ( ( ( int ) str1 [ i ] - '0' ) + ( ( int ) str2 [ i + diff ] - '0' ) ) + carry ;
    str3 = str3 + to_string ( sum % 10 ) ;
    carry = sum / 10 ;
  }
  foriinrange ( n2 - n1 - 1 , - 1 , - 1 , - 1 ) {
    int sum = ( ( ( int ) str2 [ i ] - '0' ) + carry ) ;
    str3 = str3 + to_string ( sum % 10 ) ;
    carry = sum / 10 ;
  }
  if ( ( carry ) && ( carry != 0 ) ) str3 += to_string ( carry + '0' ) ;
  str3 = str3 . substr ( 0 , str3 . length ( ) - 1 ) ;
  return returnstr3 ;
}
